### DISCUSSION TITLE
Unable to run a container

### DISCUSSION THREAD

**Discussion by mipruch** [Dokploy/dokploy#993] [Q&A]
I have no problem building the app. It is a simple "Hello world" Nuxt app (node) pulled from my GitHub. Left as is, it should natively runs on port 3000.

After the deployment is done with successful log:
<img width="572" alt="Screenshot 2024-12-25 at 21 17 12" src="https://github.com/user-attachments/assets/9a9795bf-a6fa-48f0-9d15-ee70abfe98f7" />

the app is not running. 

I'm using the default Nitpicks setting, no custom Dockerfile.

I can see the image being made using "docker images", but "docker ps -a" shows no new container.

Also, I'm unable to see the logs for the app as no container is running. 
It only shows:
Error response from daemon: No such container: select-a-container
<img width="1053" alt="Screenshot 2024-12-25 at 21 20 55" src="https://github.com/user-attachments/assets/b73de14b-fd8a-4ce1-b9f3-53e8609f5b9f" />

If I click on the Reload button, nothing happens and the spinner won't stop spinning...
If I run the image manually using "docker run -p 3010:3000 test-nuxt-pew2aa", the app runs correctly...

Any help? Thanks.

**Comment by Siumauricio** [comment]
When the error is not shown anywhere, there can be several reasons, to see the errors in more detail you can go to `/dashboard/swarm` and see the services there you should see the error in more detail.


you probably added some invalid volume, some already busy port or something else that is preventing docker from starting the container.
![image](https://github.com/user-attachments/assets/49383fc8-d549-435d-b848-29f051d7354c)

**Comment by mipruch** [comment]
<img width="738" alt="Screenshot 2025-01-03 at 2 07 12" src="https://github.com/user-attachments/assets/e1a85bf7-38ee-4c7e-bf26-371b1d8b3dd3" />
<img width="337" alt="Screenshot 2025-01-03 at 2 08 59" src="https://github.com/user-attachments/assets/8c68bf0b-9dba-4f5c-9103-d1df3018908c" />

It might be the port issue. I've completely reinstalled docker and dokploy and I'm keep getting the error. I port forward 3004 to 3000 and the 3004 port is free for sure. If I run the image manually with that port forwarding, it works fine. 

Any ideas why the Swarm throws this error? I don't use any volume mounting.

**Reply by Siumauricio** [reply]
Can you enter to your server and run docker --version?

Looks like this bug was introduced in a 27.0.2 version https://github.com/moby/moby/issues/48069

**Reply by mipruch** [reply]
It says Docker version 27.4.1, build b9d17ea.
Thanks for the link, I'll check it. What I don't quite understand is that if I create a clean EC2 instance and do exactly same steps, it works fine. But not on my local linux server.

**Reply by Siumauricio** [reply]
Is strongly recommended to install dokploy on a fresh machine to prevent kind of that issues

**Reply by mipruch** [reply]
I'm getting the same error even with Docker version 26. I'll try reinstalling linux next.
